src/EventStore.Client.Operations/EventStoreOperationsClientServiceCollectionExtensions.cs

```diff
@@ -29,7 +42,24 @@ public static class EventStoreOperationsClientServiceCollectionExtensions {
 				options.ConnectivitySettings.Address = address;
 				options.CreateHttpMessageHandler = createHttpMessageHandler;
 			});
+#endif
 
+#if NETCOREAPP3_1
+		/// <summary>
+		/// Adds an <see cref="EventStoreOperationsClient"/> to the <see cref="IServiceCollection"/>.
+		/// </summary>
+		/// <param name="services"></param>
+		/// <param name="address"></param>
+		/// <param name="createHttpMessageHandler"></param>
+		/// <returns></returns>
+		/// <exception cref="ArgumentNullException"></exception>
+		public static IServiceCollection AddEventStoreOperationsClient(this IServiceCollection services, Uri address,
+			// ReSharper disable once MethodOverloadWithOptionalParameter
+			Func<HttpMessageHandler>? createHttpMessageHandler = null)
+			=> services.AddEventStoreOperationsClient(options => {
+				options.ConnectivitySettings.Address = address;
+			});
```

Comment:
@thefringeninja Is `options.CreateHttpMessageHandler = createHttpMessageHandler;` missing here or is it intentional?

Reply:
It is intentional as grpc.core won't use it. I left the constructor parameter in there for binary compatibility.